coremltools/converters/nnssa/coreml/ssa_converter.py

```diff
@@ -184,7 +188,7 @@ def ssa_convert(ssa,
         else:
             builder.set_class_labels(classes)
 
-    image_format = ssa.get_image_format()
+    image_format = image_format or ssa.get_image_format()
```

Comment:
So is the user-provided `image_format` has a higher priority? Do we have any special handling when user-provided and SSA inferred has different image format?

Reply:
I can add a warning if the SSA infers a different format than the one provided.